extensions/functions_datetime.yaml

```diff
@@ -265,3 +347,278 @@ scalar_functions:
           - name: y
             value: interval_year
         return: boolean
+  -
+    name: "assume_timezone"
+    description: >-
+      Convert local timestamp to UTC-relative timestamp_tz using given local time's timezone.
+
+      Timezone strings must be as defined by IANA timezone database (https://www.iana.org/time-zones).
+      Examples: "Pacific/Marquesas", "Etc/GMT+1".
+      If timezone is not present then UTC is assumed. If timezone is invalid an error is thrown.
+    impls:
+      - args:
+          - name: x
+            value: timestamp
+          - name: timezone
```

Comment:
It's unclear to me why a timezone needs to be specified here.

Reply:
Because we are transforming local time to a UTC time. We need to know what the offset between the local time and UTC is.

Reply:
Ok.  So if the local time is 2:00 PM then we will store 2:00 PM UTC.  If they specify "America/Los_Angeles" then we will not store 2:00 PM UTC but we will store whatever UTC is required to get 2:00 PM in LA.  Thanks for the reminder.